### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Publish3r** [Seji64/SniDust#37]
Hi.

I'm looking for an alternative to https://github.com/ab77/netflix-proxy/ because it's outdated and didn't work anymore.
I think your SniDust could replace it.
But before i want to try it, i have a question.
In your setup guide you write that it's possible to use custom domains. That's good, i need that feature. But is it also possible to deactivate the pre-installed provider-domains?

Best regards, Publish3r.

**Comment by Publish3r** [comment]
If i would clone your project and delete the provider files. Would it work or would it cause an error at pulling / start the docker?
That's an idea for a workarround.

**Comment by Seji64** [comment]
Just delete the content of the domains.d folder - that should work

**Comment by Publish3r** [comment]
I forked your Project and changed the important parts.
I made the Github Actions and the Package was created.
I installed docker on a Hetzner Test VPS, solved the Port 53 issue and pulled / ran the container.
So far so good.
It's working.
There is only one problem.
The DynDNS part isn't working. 
After i restarted the Container with Plain IP SniDust is working.
If i use DynDNS only it's not working.
My DynDNS Provider is goip.de should i use a different one?

---

Achja, wir können auch gerne auf deutsch schreiben. Ich hasse es, alles in schlechtes Englisch übersetzen zu müssen. lol
